website/blog/2022-09-05-version-070.md

```diff
@@ -0,0 +1,111 @@
+
+Autolinking it's a crucial part of the React Native development experience. It allows you to include external libraries with a `yarn add` command without having to deal with CocoaPods or Gradle setups.
+
+The New Architecture required us to adapt the auto-linking features to support libraries which are using the Codegen and exposing native code to app developers.
+
+While Autolinking worked well for New Architecture libraries on iOS, the same was not true for Android. With 0.70 we closed this gap and you can now keep on including libraries with `yarn add` to your project, they will be linked correctly both on any architecture.
+
+## Full CMake support for Android builds
+
+Starting from 0.70, users can now use CMake to configure their Native builds. While we don’t expect app users to directly write C++ code, you still need an entry point for the native compilation.
+
+From now on you can use a `CMakeLists.txt` file instead of an `Android.mk` file for anything Android/Native related in your project.
+
+This change will benefit app and library users on the New Architecture as:
+
+- The CMake file created in your app is way smaller ([3 lines of code](https://github.com/facebook/react-native/blob/9923ac1b524ae959abdf50a28a3094198015f77e/packages/rn-tester/android/app/src/main/jni/CMakeLists.txt#L6-L11) versus [50+ for Android.mk files](https://github.com/facebook/react-native/blob/main/template/android/app/src/main/jni/Android.mk?rgh-link-date=2022-07-20T18%3A29%3A07Z)) so this makes for an easier update experience between React Native versions in the future and less code to maintain on your end.
+- Codegen is now generating both `Android.mk` and `CMakeLists.txt`, so libraries should not worry about doing anything if they're using the default setup we provide for New Architecture libraries.
+- The Auto-linking mentioned above will work with both CMake and Android.mk files out of the box.
+- Despite apps being free to use either `Android.mk` or CMake files, the recommended solution in the future would be CMake files (due to better documentation, tooling and ecosystem around CMake).
+
+## Highlights of 0.70
+
+As mentioned above, some of the more important improvements in this release are centered around the New Architecture experience. However, there have been other notable changes, including:
+
+- Fix for Catalyst is live, set mac_catalyst_enabled to true in Podfile (see [upgrade-helper](https://react-native-community.github.io/upgrade-helper/?from=0.69.1&to=0.70.0-rc.0) diff for details).
+- Bumping metro to 0.72.0 should enable the "new" react jsx transform: [reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html](https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html).
+- Removing reactnativeutilsjni as it is built from the same sources as reactnativejni ([https://github.com/facebook/react-native/pull/34339](https://github.com/facebook/react-native/pull/34339)).
```

Comment:
```suggestion
- Removing `reactnativeutilsjni` as it is built from the same sources as `reactnativejni` ([https://github.com/facebook/react-native/pull/34339](https://github.com/facebook/react-native/pull/34339)).
```

Reply:
not sure about this. Are pieces of code or folders?

Reply:
They're module names so `<code>` is fine here